orchagent/main.cpp

```diff
@@ -104,10 +104,10 @@ int main(int argc, char **argv)
 
     int opt;
     sai_status_t status;
-
+    bool gMacAgingTimeSet = false;
```

Comment:
for this attribute, i think there's no need to make it global.

Reply:
Hi Stcheng, thanks for review, yes there is no need to make it global. I just named it wrongly.